web/src/app/home/HomeComponent.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+
+class HomeComponent extends Component {
```

Comment:
Why two lines between imports and component definition? Lets push for consistancy in style across frontend code.